src/Poke-API/api.jsx

```
@@ -62,44 +65,83 @@ const PokeApi = () => {
   const getStatColor = (value) => {
-    if (value <= 50) return 'success'; // Hijau (lemah)
-    if (value <= 80) return 'warning'; // Kuning (lumayan kuat)
-    if (value <= 100) return 'danger'; // Oranye/Merah (kuat)
-    return 'danger'; // Merah (sangat kuat)
+    if (value <= 50) return 'success'; // Green (easy)
+    if (value <= 80) return 'warning'; // Yellow (medium)
+    if (value <= 100) return 'danger'; // Orange (hard)
+    return 'danger'; // Red (superpower)
   };
 
+  const startIndex = (page - 1) * limit;
```

ESLint

[error] 'startIndex' is assigned a value but never used.

(no-unused-vars)